README.md

```diff
@@ -25,6 +25,8 @@ composer require jblond/php-diff
 
 ## Example Use
 
+<details><summery>&nbsp;</summery><br>
```

Comment:
There's a typo in the `<summary>` tags.
Are you sure you want the content to be a non breaking space instead of a description?

Reply:
Nope, I guess it didn't work in the first place because of the typo.